docs/spec/v2/helmreleases.md

```diff
@@ -494,6 +494,8 @@ The field offers the following subfields:
   rendered templates against the Kubernetes OpenAPI Schema. Defaults to `false`.
 - `.disableSchemaValidation` (Optional): Prevents Helm from validating the
   values against the JSON Schema. Defaults to `false`.
+- `.disableTakeOwnership` (Optional): Disables ignoring the check for helm labels and annotations before taking
+  ownership of the existing resources during the Helm upgrade action. Defaults to `false`.
```

Comment:
I think the presence of `ignoring the check for helm labels and annotations before` makes the documentation confusing here. This thing about labels and annotations is just how Helm implements the ownership control so it's not super relevant here.

```
- `.disableTakeOwnership` (Optional): Disables taking ownership of existing resources during the Helm <insert action here> action. Defaults to `false`.
```